tests/API/XCCDF/unittests/test_remediation_simple.sh

```diff
@@ -43,4 +45,19 @@ assert_exists 1 '//score'
 assert_exists 1 '//score[text()="100.000000"]'
 
 rm test_file
-rm $result
+
+$OSCAP xccdf generate fix --fix-type bash --profile '' --output "$result" "$input_xml" 2> $stderr
```

Comment:
We should generate remediation role for specific Profile.
I guess this only works because Rule `xccdf_moc.elpmaxe.www_rule_1` is selected by default.

Reply:
It is probably fitting into `default` Profile, but than it is strange that is outputs title and description for Profile `xccdf_moc.elpmaxe.www_profile_common`

Reply:
If there is only one profile then "" will only match one profile and it will all work. I agree that listing the profile or at least the "common" suffix would improve readability here.

Reply:
I have added another profile to the XML as suggested by @yuumasato and modified the oscap call. I don't see the reasons why to add another test - I guess that the test is still pretty minimalistic.